### PR TITLE
Refactor for Swift 3 Release

### PR DESCRIPTION
**Pull request by RafaelPlantard**
Removed all warnings and errors generated on Xcode 8. Reference: #18

### CODE REVIEW

File: Randoms.swift [hunk 1]
```diff
@@ -18,91 +18,88 @@ public extension Bool {
...
+    public static func random(_ lower: Int = 0, _ upper: Int = 100) -> Int {
         return lower + Int(arc4random_uniform(UInt32(upper - lower + 1)))
     }
 }
 
 public extension Int32 {
     /// SwiftRandom extension
-    public static func random(range: Range<Int>) -> Int32 {
         return random(range.upperBound, lower: range.lowerBound)
+    public static func random(_ range: Range<Int>) -> Int32 {
```
Comment (thellimist) [inline Randoms.swift]: This is so wrong. Please test the code before sending a PR

Comment (thellimist) [inline Randoms.swift]: http://pastebin.com/gVeUXbKb

here is a simple test case. Please test everything before sending a PR

Comment (RafaelPlantard) [inline Randoms.swift]: @thellimist Let me try understand your point, is there any problem in implementation or just you want unit tests?

Comment (RafaelPlantard) [inline Randoms.swift]: @thellimist I'm so sorry, I forget to commit all changes before sent you a PR, really I'm so sorry.

**Pull request merged**